interface/ceed-vector.c

```diff
@@ -220,6 +220,43 @@ int CeedVectorReferenceCopy(CeedVector vec, CeedVector *vec_copy) {
   return CEED_ERROR_SUCCESS;
 }
 
+/**
+  @brief Copy a CeedVector into a different CeedVector.
+          Both pointers should be destroyed with `CeedVectorDestroy()`.
+          Note: If `*vec_copy` is non-NULL, then it is assumed that `*vec_copy` is a pointer to a CeedVector.
+          This CeedVector will be destroyed if `*vec_copy` is the only reference to this CeedVector.
+
+  @param[in]     vec      CeedVector to copy
+  @param[in,out] vec_copy Variable to store copied CeedVector to
+
+  @return An error code: 0 - success, otherwise - failure
+
+  @ref User
+**/
+int CeedVectorCopy(CeedVector vec, CeedVector *vec_copy) {
```

Comment:
I think this is throwing you here:
```suggestion
int CeedVectorCopy(CeedVector vec, CeedVector vec_copy) {
```

(This change will cause other changes)